src/rustdoc.md

```diff
@@ -2,6 +2,8 @@
 
 Rustdoc actually uses the rustc internals directly. It lives in-tree with the
 compiler and standard library. This chapter is about how it works.
+For information about Rustdoc's features and how to use them, see
+the [Rustdoc book](https://doc.rust-lang.org/nightly/rustdoc/).
```

Comment:
Any reason why you linked to *nightly* rustdoc?

Reply:
Yes: this is the dev-guide, which means the people reading it are contributors and interested in the latest version, which is the one they'll be working on. If you're not a contributor and on the wrong page it's easy enough to change the URL.

Reply:
Sounds good :)